### PR TITLE
Comma-separate multiple names in a @param directive

### LINKED ISSUE #131: list of params renders without commas
```
@param r,g,b The color components to use
```

renders as `rgb` instead of `r, g, b`. I've never used multiple variables in one `@param` before so this is probably why it doesn't work.

### PR DESCRIPTION
**Pull request by evanlenz**
fix #131

### CODE REVIEW

File: include/docca/base-stage1.xsl [hunk 1]
```diff
@@ -320,17 +320,21 @@
           <xsl:template mode="parameter-row" match="parameteritem">
             <tr>
               <td>
-                <code>
-                  <!-- ASSUMPTION: <parameternamelist> only ever has one <parametername> child -->
```
Comment (evanlenz) [inline include/docca/base-stage1.xsl:324]: @vinniefalco It turned out to be a bad assumption, but at least I felt sufficiently unsettled about it to document it! 😂

Comment (vinniefalco) [reply include/docca/base-stage1.xsl:324]: lmao... :)

**Comment by vinniefalco** [comment]
Is "issue132" in the table of contents?

**Comment by vinniefalco** [comment]
OH 131

**Comment by vinniefalco** [comment]
BRO.... you don't understand how good this is

**Comment by evanlenz** [comment]
> BRO.... you don't understand how good this is

Glad you like it, bro! 😁

**Pull request merged**